### PR TITLE
Chuck Norris fact bot

### PR DESCRIPTION
**Pull request by MBahrSNC**

### CODE REVIEW

**Comment by jacebenson** [comment]
Looks like this gets #29

File: responses/chuck-norris.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
+module.exports = {
+    command: function(bot, msg) {
+console.log('in cat command');
```
Comment (jacebenson) [inline responses/chuck-norris.js]: lets comment or leave this out

File: responses/chuck-norris.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+        var phrase = '!chuck';
+console.log('msg.author.bot', msg.author.bot);
```
Comment (jacebenson) [inline responses/chuck-norris.js]: lets comment or leave this out

File: responses/chuck-norris.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+        if (msg.author.bot === false) {
+            var wordsArr = msg.content.split(' ');
+console.log('wordsArr',wordsArr);
```
Comment (jacebenson) [inline responses/chuck-norris.js]: lets comment or leave this out

**Comment by MBahrSNC** [comment]
Oh. Yup. My bad.

**Pull request merged**